piker/data/feed.py

```diff
@@ -108,7 +116,11 @@ def get_feed_bus(
     return _bus
 
 
-async def _setup_persistent_brokerd(brokername:  str) -> None:
+@tractor.context
```

Comment:
This was the main new bit: show's how we setup `brokerd` state using `tractor`'s new `Context` api.